pkg/descheduler/setupplugins.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Does this line need to be updated?

Reply:
This is copied from k8s descheduler, it's better to keep original format?

Reply:
ok

Reply:
We need to keep it as is, since respecting the original copyright is required by Apached 2.0 license.

Reply:
great!